### PR TITLE
Add dish dialog

### PR DESCRIPTION
**Pull request by lei29**
Add a button that pop up a dialog to add a new dish. run the frontend and go to chef/edit/1 to view the page

### CODE REVIEW

File: client/src/components/AddDishDialog.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <DialogTitle id="form-dialog-title">New Dish</DialogTitle>
+        <DialogContent>
+          <DialogContentText>
+            To add a dish, please fill in the form and click submit button.
+          </DialogContentText>
+          <TextField
+            autoFocus
+            margin="dense"
+            id="serve"
+            label="How many people will this dish serve."
```
Comment (smithpg) [inline client/src/components/AddDishDialog.js]:
```suggestion
            label="How many people will this dish serve?"
```

File: client/src/components/DishCard.js [hunk 2]
```diff
@@ -20,28 +26,150 @@ const useStyles = makeStyles({
...
     display: 'flex',
     flexDirection:"column",
-    padding:40,
+    paddingLeft:40,
+    paddingTop:40,
+    paddingBottom:40,
+  },
+
+  pplnum:{
+    backgroundColor: "#FF743D",
```
Comment (smithpg) [inline client/src/components/DishCard.js]: We should probably stick to importing the brand color from themes/theme.js

File: client/src/components/DishCard.js [hunk 3]
```diff
@@ -20,28 +26,150 @@ const useStyles = makeStyles({
...
+  dishpic:{
+    cursor:"pointer",
   }
+
 });
 
 export default function SimpleCard() {
   const classes = useStyles();
 
+  const handleChange = name => event => {
```
Comment (smithpg) [inline client/src/components/DishCard.js]: You can use one function that gets the name from an attribute on the target input element for this. First we'd need to add name attributes to each input (e.g. price input would get name="price"), then modify handleChange to look like:

handleChange = event => {setValues({...values, [event.target.name] : event.target.value});

File: client/src/components/DishCard.js [hunk 4]
```diff
@@ -20,28 +26,150 @@ const useStyles = makeStyles({
...
         <Grid className={classes.info}>
-            <span id="pplnum"> MEAL FOR 2</span>
-            <span id="dishname"> 4 specialty rolls </span>
-            <span id="price"> $15.00 </span>
-            <span id="upperbig">INGREDIENTS</span>
-            <p id="greysmall">Rice, nori, avacado, crab, cucumber, wasabi, rice vinegar, soy sauce, salt, sugar.</p>
-            <span id="upperbig">REQUIRED STUFF</span>
-            <p id="greysmall">Kitchen Table, Cooking plate</p>
+          <TextField
+              id="outlined-name"
```
Comment (smithpg) [inline client/src/components/DishCard.js]: Are the id's on these TextFields necessary?

File: client/src/components/NameCard.js [hunk 5]
```diff
@@ -21,30 +24,158 @@ const Container = styled.div`
...
+                    <img className={classes.cover} alt="background" src="/cover-sushi.png" />
+
+                </label>
+                <input
+                    accept="image/*"
+                    className={classes.input}
+                    id="profile-img-file"
+                    multiple
+                    type="file"
+                />
```
Comment (smithpg) [inline client/src/components/NameCard.js:107]: We should add something to these two image file inputs to visually represent the fact that they can be clicked and are inputs. Right now they look the same as they do when viewing the profile in read-only mode

File: client/src/components/DishCard.js (hunk 2 above)
Comment (lei29) [reply client/src/components/DishCard.js]: I tried that, but did not figure out how to do that, ${brand } doesn't work.

File: client/src/components/NameCard.js (hunk 5 above)
Comment (lei29) [reply client/src/components/NameCard.js:107]: I will do a hover effect and show a msg like click here to upload new img.

File: client/src/components/DishCard.js (hunk 2 above)
Comment (lei29) [reply client/src/components/DishCard.js]: Fixed

**Pull request merged**